PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+### Description of work
+
+*Add your own description here. The aim is provide information to help the reviewer review the PR.*
+
+### Issue
+
+*Link to Issue, if there is one.*
+
+### Acceptance Criteria
+
+*List the changes in functionality or code that the reviewer needs to review.*
+
+### Unit Tests
+
+*Give an overview of any unit tests you have added or modified, if applicable.*
```

Comment:
Isn't this obvious in the changes? Could just have a tickbox in the "Code Review" bit for "Unit tests adequately cover changed/new code"?